src/components/ebay-progress-bar/component.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    getValue(value, min, max) {
+        const parsedValue = parseInt(value);
```

Comment:
Make sure you do `parseInt(value, 10);` (should always have the `radix` param, in case it accidentally parses as a hex or something else)